website/bin/replace-snippets.js

```diff
@@ -0,0 +1,34 @@
+const fs = require("fs");
+const path = require('path')
+const replace = require('replace-in-file')
+const snippets = require('../snippets')
+
+//shell.fatal = true; // same as "set -e"
+
+const options = {
+
+  // Glob(s)
+  files: [
+    path.join(__dirname, '/../tests/*'),
+    path.join(__dirname, '/../../docs/**/*')
+  ],
+
+  // Replacement to make (string or regex)
+  from: replacements["replacements"][i]["from"],
```

Comment:
yeah think we need to define replacements first, maybe you're about to commit this, just a reminder

Reply:
Updated!